test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Import/ImportTests.cs

```diff
@@ -92,6 +92,39 @@ public async Task GivenIncrementalLoad_MultipleInputVersionsOutOfOrderSomeNotExp
             Assert.Equal(GetLastUpdated("2002"), result.Resource.Meta.LastUpdated);
         }
 
+        [Fact]
```

Comment:
Can we add a test where we provide non sequential resource versions?

Reply:
Not sure what you mean. Input order of records is immaterial as all are sorted internally.